### DISCUSSION TITLE
Updating to Traefik 2.10.5 to fix CVE-2023-44487

### DISCUSSION THREAD

**Discussion by rome-user** [k3s-io/k3s#8622] [Q&A] [answered]
Hi,

Given that Traefik 2.10.5 has a fix for CVE-2023-44487, is it possible to release a new minor release of k3s that ships this newer release? If it is not feasible, then is there a way to look at Helm chart values k3s uses for installing Traefik before deciding to install Traefik manually in my cluster?

Thank you

**Comment by llabordehpe** [comment]
Hello @brandond,

Any plan to release a new version of k3s that includes Traefik 2.10.5 submitted in https://github.com/k3s-io/k3s/commit/3abc8b82ed0779ebaa5d0ca00165408ad085cc8f ?

**Reply by brandond** [reply, marked as answer]
Yes, it'll be included in a release later this week when we get new patch releases from upstream, same as we do every month.
